app/src/main/AndroidManifest.xml

```diff
@@ -84,6 +84,7 @@
       android:windowSoftInputMode="adjustResize" />
     <activity
       android:name=".app.profile.ProfileChooserActivity"
+      android:label="@string/profile_chooser_select"
```

Comment:
Should we maybe use a label like "Profile selection page" to have parity with other "page"-like activity labels?

Reply:
Yes its better to use "Profile Selection Page", then the flow will be like this
`Profile Selection Page`
`Select Your Profile`
`Admin, Administrator...`
`Set up Multiple Profile...`
`Administrator controls`
But don't you think that Administrator Controls is a bit odd here. Also it can only accessed by admin, So I think it is better to show it inside NavigationDrawer (Because using this feature needs a pin verification anyways)

Reply:
Actually `Admin Controls` is correct here and the `Talkback` explicitly reads `out-of-list` or `in-list` while shifting from in and out of recyclerview items and that won't create any confusion.

Reply:
Actually `Admin Controls` is correct here and the `Talkback` explicitly reads `out-of-list` or `in-list` while shifting from in and out of recyclerview items and that won't create any confusion.